src/main/webapp/app/shared/virtual-scroll/virtual-scroll.style.css

```diff
@@ -0,0 +1,19 @@
+:host {
+    display: block;
+    position: relative;
+    height: 500px;
```

Comment:
Could we make this configurable? Seems kind of arbitrary to have it always at 500px

Reply:
Done ✔️